tests/io/test_data_catalog.py

```diff
@@ -373,6 +380,20 @@ def test_config_relative_import(self, sane_config):
         with pytest.raises(DataSetError, match=re.escape(pattern)):
             DataCatalog.from_config(**sane_config)
 
+    def test_config_import_kedro_datasets(self, sane_config, mocker):
+        """Test kedro.extras.datasets default path to the dataset class"""
+        # Spy _load_obj because kedro_datasets is installed
+        class DummyMock:
```

Comment:
I think `DummyMock` and `dummy_mock` could have more meaningful names, would `DummyLoader` work here?

Reply:
@jmholzer I manage to remove the dummy object, turns out I can spy on the `kedro.io.core` module object directly, so the `patch` is not needed too.